taxcalc/policy.py

```diff
@@ -305,6 +312,26 @@ def scan_param_code(code):
             msg += code
             raise ValueError(msg)
 
+    def cpi(self, param_code_name):
```

Comment:
I think it would be good to add a unit test for this function. Also, it seems to me that the use case is specific enough that a more descriptive name should be used.